2200090/소득 불균형.py

```diff
@@ -0,0 +1,9 @@
+for t in range(1, int(input())+1):
+    n = int(input())
+    number = list(map(int, input().split()))
+    avg_ = sum(number) // len(number)
```

Comment:
```suggestion
    avg_ = sum(number) / len(number)
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄